src/server.js

```diff
@@ -299,11 +303,12 @@ class Server {
    * @param  {String}   name    name of queue
    * @param  {Promise}  worker  worker promise to run
    * @param  {Object}   job     job for worker
+   * @param  {Object}   jobMeta job metadata
    * @param  {Function} done    worker callback
    * @return {undefined}
    */
-  _enqueue (name: string, worker: Promise<*>, job: Object, headers: Object, done: Function) {
-    this._workQueues[name].push(this._runWorker.bind(this, name, worker, job, headers, done))
+  _enqueue (name: string, worker: Promise<*>, job: Object, jobMeta: Object, done: Function) {
+    this._workQueues[name].push(this._runWorker.bind(this, name, worker, job, jobMeta || {}, done))
```

Comment:
Got it. I think you are right. No need for that default because `jobMeta` is set to `{}` earlier